catroid/src/main/res/layout/dialog_new_actor.xml

```diff
@@ -104,13 +104,22 @@ android:paddingBottom="@dimen/dialog_content_area_padding_top">
         <TableRow
             android:layout_width="match_parent"
             android:layout_height="match_parent">
+
              <TextView
                  android:id="@+id/dialog_new_look_from_local"
                  android:background="@drawable/new_object_dialog_selector"
                  android:gravity="center"
                  android:text="@string/add_look_from_local_project"
                  android:textSize="?attr/medium"
                  app:drawableTopCompat="@drawable/ic_library_add" />
+
+            <TextView
+                android:id="@+id/dialog_new_look_empty_object"
+                android:background="@drawable/new_object_dialog_selector"
+                android:gravity="center"
+                android:text="@string/add_look_empty_object"
+                android:textSize="?attr/medium"
+                app:drawableTopCompat="@drawable/ic_plus" />
```

Comment:
Please add a drawableTint, because the Icon seems to be light blue in comparison with the others being white

e.g. app:drawableTint="@color/solid_white"